Gems/Atom/Feature/Common/Assets/Shaders/Materials/EnhancedPBR/EnhancedPBR_Lighting.azsli

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+// This #define magic lets you use the EvaluateLighting function in this file without making it the final EvaluateLighting
+// used in your shader. Simply #define EvaluateLighting to your custom definition before including this file
+//
+#ifndef EvaluateLighting
+#define EvaluateLighting(surface, screenPosition)       EvaluateLighting_StandardPBR(surface, screenPosition)
+#endif
+
+#include "../BasePBR/BasePBR_Lighting.azsli"
+
+void InitializeLightingData_StandardPBR(Surface surface, float4 screenPosition, inout LightingData lightingData)
+{
+    // --- Base PBR ---
+    InitializeLightingData_BasePBR(surface, screenPosition, lightingData);
+
+    // --- Standard PBR ---
+
+    lightingData.emissiveLighting = surface.emissiveLighting;
+    lightingData.diffuseAmbientOcclusion = surface.diffuseAmbientOcclusion;
+    lightingData.specularOcclusion = surface.specularOcclusion;
+
+    if(o_clearCoat_feature_enabled)
+    {
+        // Clear coat layer has fixed IOR = 1.5 and transparent => F0 = (1.5 - 1)^2 / (1.5 + 1)^2 = 0.04
+        lightingData.diffuseResponse *= 1.0 - (FresnelSchlickWithRoughness(lightingData.NdotV, float3(0.04, 0.04, 0.04), surface.clearCoat.roughness) * surface.clearCoat.factor);
+    }
+}
+
+void HandleOpacity(Surface surface, inout LightingData lightingData)
```

Comment:
Actually this whole file can go, already use StandardPBR_Lighting in EnhancedPBR.azsli
With regards to the code living in lighting, most of this code does modify DiffuseLighting and SpecularLighting, and opacity handling is something that is material specific (BasePBR doesn't do opacity and StandardPBR won't in the future either), so can't be moved to ForwardPassVertexAndPixel

Reply:
So I think what we actually need is another file "TransparentPassVertexAndPixel". I am expecting we will separate the forward and transparent pass into separate shader assets (per https://github.com/o3de/sig-graphics-audio/issues/74), instead of hijacking the forward pass shader asset. Then we could move HandleOpacity there.

So I guess this can wait for a separate PR where we change the way transparency is handled.